products/windows-server.md

```diff
@@ -130,6 +130,22 @@ releases:
     latest: 6.0.6003
     link: https://learn.microsoft.com/lifecycle/products/windows-server-2008
 
+-   releaseCycle: "2003"
```

Comment:
The order of those releases is not right (that's what makes the build fail). Releases must be ordered by release date (from the most recent to the oldest).

Reply:
@marcwrobel i have done that for you , could you make a fast review , i have also fixed sp1's release date according to chatgpt